src/types.ts

```diff
@@ -205,7 +205,7 @@ export interface PhaseConfig<
   CtxWithPlugins extends Ctx = Ctx
 > {
   start?: boolean;
-  next?: string;
+  next?: ((G: G, ctx: CtxWithPlugins) => any) | string;
```

Comment:
This is good, but we can be more precise like so:

```suggestion
  next?: ((G: G, ctx: CtxWithPlugins) => string | void) | string;
```

This says that a `next` method should return a string or nothing (`void` is basically like `undefined`) so if people try `next: () => 5` or whatever, they’ll see errors.

Reply:
I get the thinking and I agree - however, this causes failures elsewhere because the `Ctx` interface expects `phase` to be of type `string`. I've tried updating the expected type to `string | void` with no luck :(

Any guidance would be appreciated.

Reply:
Oh, right, but that’s actually good because it’s telling us there’s a bug. `phase` should be `null` when there is no phase, but we’d be setting it to `undefined`. If we set phase to `phaseConfig.wrapped.next(state) || null` this type change will work.

Reply:
Ah, makes sense. I had to also update default wrapped function to match.